Modelica/Electrical/Analog/Lines/Functions/TestLineCmatrix.mo

```diff
@@ -0,0 +1,76 @@
+within Modelica.Electrical.Analog.Lines.Functions;
+model TestLineCmatrix
```

Comment:
I would prefer to leave these two tests there, since here we just test local functions, not executable models.

The directory of Modelica.Analog.examples is already too cluttered (a few years ago I got accepted the use of a new subfolder "Lines", wich mitigates the issue).
IMO it is better to put in the examples folder tests of simulation models, while it is clearer for users to let these direct tests of functions, near to them.
Can this be accepted?

Reply:
I think there is no case in MSL having a test outside of an Examples package. 

Is the test usable for users to whatever check? If so, it could be defined being a test *function* instead. Moreover, it seems there is indeed no time simulation needed in order to provide the prints. So a function could fit well.

Ditto for TestLineZmatrix

Reply:
I try to explain.
The provided functions are intended for two purposes
1. be used at the beginning of a simulation to compute C and Z matrices from conductor geometry; the matrices are in turn used later for simulation
2. just to find the matrices from conductors geometry.

The purpose 1 is demonstrated in PowerLineWithFence (and probably by a second future example in case I create a new version of M_Oline to overcome its limitations).
The purpose 2 is demostrated by TestZmatrix and TestCmatrix. For this, the user has just to set the geometry in record g (stands for "geometry". This is very convenient. Consider that some tools (at least OpenModelica, which is very important since it is totally free) do not allow running stand-alone functions. 

So, since it seems to me that leaving TestCmatrix and TestZmatrix as models in the position where I put them, I can move where you recommended initially, i.e. in Examples.Lines. This for me, even though suboptimal, is much more acceptable than converting them into functions.
Is this acceptable for you?

Reply:
I have just concern about MSL consistency.
@HansOlsson @beutlich Any idea?